magento/magento-cloud-docker
# MAGECLOUD-5185: Platform-agnostic Docker builder #102

### Description


### Fixed Issues (if relevant)

1. #102
2. https://magento2.atlassian.net/browse/MAGECLOUD-5185

### Manual testing scenarios

!!! Tests should be executed with `--with-xdebug` option till the appropriate bug is fixed:

1. https://jira.corp.magento.com/browse/MAGECLOUD-66
2. https://jira.corp.magento.com/browse/MAGECLOUD-116

### Contribution checklist
